Makefile

```diff
@@ -0,0 +1,12 @@
+run:
+	cargo run $(path)
+build:
+	cargo build
+test:
+	cargo test
+clean:
+	rm -r ./target
+2048:
+	cargo run ./binaries/2048.obj
+rogue:
+	cargo run ./binaries/rogue.obj
```

Comment:
Always put a new line at the end of the files. You can configure your editor to do it automatically